### PR TITLE
Extract findActivities() to DataManipulateDao

### PR DESCRIPTION
**Pull request by Billy4195**
Extract the findActivities() to DataManipulateDao, and use this function to replace the SQL in find_image_acticity()

### CODE REVIEW

File: dataAccessObjects.py [hunk 1]
```diff
@@ -175,44 +215,6 @@ def insertUndecidedSchedule(self,targetId,displayTime,arrangeModeSn):
...
-                +" ({type_condition})".format(type_condition=type_condition)\
-                +" and text_is_delete=0 and text_is_expire=0 "\
-                +" and (TO_DAYS(NOW()) between TO_DAYS(text_start_date) and TO_DAYS(text_end_date)) " \
-                +" and (TIME_TO_SEC(DATE_FORMAT(NOW(), '%H:%i:%s')) between TIME_TO_SEC(text_start_time) and TIME_TO_SEC(text_end_time))) AS a0 "\
-                +" LEFT JOIN (SELECT type_id, type_weight FROM data_type WHERE "\
-                +" ({type_condition})".format(type_condition=type_condition)\
-                +" ) AS a1 ON a0.type_id=a1.type_id ORDER BY a1.type_weight ASC"
-        ret = self.db.query(sql)
-        return ret
-
```
Comment (stvreumi) [inline dataAccessObjects.py]: Why don't I see adding `super()` in `TextDao` and `ImageDao` in this commit?

Comment (icecat2012) [inline dataAccessObjects.py]: I test it a few minutes ago.
It was provide by heir class.
So, it is correct that no super() in `TextDao` and `ImageDao`.

Comment (stvreumi) [inline dataAccessObjects.py]: Magic...

Comment (Billy4195) [inline dataAccessObjects.py]: Because the derived class can use the method from the parent class

**Comment by Billy4195** [comment]
@stvreumi @chenyang14 @icecat2012  
I solved the conflict and push again, please help me to review it.

**Pull request merged**